ansible/playbooks/restart-node.yml

```diff
@@ -0,0 +1,32 @@
+---
+- hosts: '{{ inventory }}'
+  serial: "{{ stride | default(10) }}"
+  gather_facts: yes
+
+  tasks:
+  - name: gather OS specific variables
+    include_vars: "{{ item }}"
+    with_first_found:
+      - '{{ ansible_bios_version }}-{{ ansible_distribution }}.yml'
+      - '{{ cloud_platform }}-{{ ansible_distribution }}.yml'
+    tags:
+      - vars
+
+  - name: restart node
+    become: true
+    become_user: root
```

Comment:
it is no need for setting `become_user` param. the default become user for `become: true` is root.